extensions-core/google-extensions/pom.xml

```diff
@@ -153,7 +153,7 @@
             <groupId>org.apache.druid</groupId>
             <artifactId>druid-server</artifactId>
             <version>${project.parent.version}</version>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
Same comment as above - compile doesn't seem right to me.

Reply:
Changed